pkg/sqlqueries/SQL/trb_request_system/get_by_trb_request_id_LOADER.sql

```diff
@@ -0,0 +1,16 @@
+WITH queried_trb_request_ids AS (
+    SELECT trb_request_id AS trb_request_id
+    FROM JSON_TO_RECORDSET(:param_table_json)
+        AS x("trb_request_id" UUID) --noqa
+
+)
+SELECT
+    sis.id,
+    sis.trb_request_id,
+    sis.system_id,
+    sis.created_by,
+    sis.created_at,
+    sis.modified_by,
+    sis.modified_at
+FROM queried_trb_request_ids qsid
```

Comment:
super super nit pick: change table alias from `qsid` to something like `qtid`

Reply:
Addressed in [ca20523](https://github.com/CMSgov/easi-app/pull/2445/commits/ca20523b290c8a45960d581688769f1feb3c66a8)